src/schema/v1/sale_artwork.ts

```diff
@@ -370,6 +371,31 @@ export const SaleArtworkType = new GraphQLObjectType<any, ResolverContext>({
           return saleLoader(sale_id)
         },
       },
+      calculated_cost: {
+        type: CalculatedCostType,
+        args: {
+          bid_amount_cents: {
+            type: GraphQLInt,
+            description: "Max bid price for the sale artwork",
+          },
+        },
+        resolve: (_params, { bid_amount_cents }, _loaders) => {
+          return {
+            buyers_premium: {
+              cents: bid_amount_cents * 0.2,
+              display: `$${((bid_amount_cents * 0.2) / 100)
+                .toFixed(2)
+                .replace(/\d(?=(\d{3})+\.)/g, "$&,")}`,
+            },
+            subtotal: {
+              cents: bid_amount_cents * 1.2,
+              display: `$${((bid_amount_cents * 1.2) / 100)
+                .toFixed(2)
+                .replace(/\d(?=(\d{3})+\.)/g, "$&,")}`,
```

Comment:
Just hard-coding the amount and doing a simple math so the client can actually see that the value gets updated based on the argument, but I'm not really trying to optimize this code since we will replace it soon.

Reply:
Small thing: to save the reader needing to look up the definition of this regex, thoughts on extracting a named variable for it? I think it would make the code a bit more clear.

Reply:
The next time we touch this code we will probably just remove it entirely rather than understanding what it does. All we have to know is it does some kind of formatting to make fake data close to reality, and I don't think it is worth extracting.

![AUCT-671](https://user-images.githubusercontent.com/386234/67121259-167c4f00-f1b9-11e9-8d91-2dad6ac848ba.png)

Reply:
Fair enough!